framework/python/src/common/mqtt.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""MQTT client"""
+import typing as t
+import paho.mqtt.client as mqtt_client
+from common import logger
+from common import util
+
+LOGGER = logger.get_logger("runner")
```

Comment:
Could we rename the logger to something like mqtt_client?

Reply:
Renamed